atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/knowledge/KnownAbortedTransactionsImpl.java

```diff
@@ -99,7 +99,8 @@ public boolean isKnownAborted(long startTimestamp) {
 
     @Override
     public void addAbortedTimestamps(Set<Long> abortedTimestamps) {
-        futileTimestampStore.addAbortedTimestamps(abortedTimestamps);
+        // todo(snanda): batching?
```

Comment:
let's just make sure this is tracked somewhere

Reply:
Will do